pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1379,10 +1397,14 @@ func doAssignIoAdaptersToDomain(ctx *domainContext, config types.DomainConfig,
 					ib.Phylabel, ib.UsbAddr, status.DomainName)
 				assignmentsUsb = addNoDuplicate(assignmentsUsb, ib.UsbAddr)
 			} else if ib.PciLong != "" && !ib.IsPCIBack && !ib.KeepInHost {
-				log.Functionf("Assigning %s (%s) to %s",
-					ib.Phylabel, ib.PciLong, status.DomainName)
-				assignmentsPci = addNoDuplicate(assignmentsPci, ib.PciLong)
-				ib.IsPCIBack = true
+				if !(ctx.hvTypeKube && config.VirtualizationMode == types.NOHYPER) || ib.Type != types.IoNetEth {
+					log.Functionf("Assigning %s (%s) to %s",
+						ib.Phylabel, ib.PciLong, status.DomainName)
+					assignmentsPci = addNoDuplicate(assignmentsPci, ib.PciLong)
+					ib.IsPCIBack = true
+				} else {
+					log.Noticef("doAssignIoAdaptersToDomain: skip IO assign %v", ib)
```

Comment:
Would be good to return an error to the user by using `ib.Error` - isn't it possible here?

Reply:
but this is not an error, it's only we skip handling the pass through if in that condition, we'll have another mechanism to use this io bundle later in domainmgr.

Reply:
Okay. Can you please add a comment in the code about this other mechanism?

Reply:
@naiming-zededa waiting for your response here.

Reply:
ok, @zedi-pramodh please add comment in this place:
// for native container with ethernet IO passthrough, we uses the NAD for the Multus
// for the container to directly access the ethernet port through network mechanism

Reply:
ack, this change will be in next commit